.github/ISSUE_TEMPLATE/issueForm.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug Report
```

Comment:
We should also have the ability for them to submit a feature request. We should make another form for that requesting

1. Feature information
2. Example functionality
3. Links to relevant docs